app/tests/test_search.py

```diff
@@ -2274,18 +2274,11 @@ def test_search_transferring_body_no_highlights_renders_file_name_and_search_res
         assert response.status_code == 200
         soup = BeautifulSoup(response.data, "html.parser")
         table_body = soup.find("tbody")
-        rows = table_body.find_all("tr")
-        # Find the row with file name
-        found = False
-        for row in rows:
-            cells = row.find_all("td")
-            if len(cells) == 2 and "File name" in cells[0].get_text():
-                link = cells[1].find("a")
-                assert link
-                assert file_name in link.get_text()
-                assert url_for("main.record", record_id=file_id) in link["href"]
-                found = True
-        assert found
+        table_rows_cell_values = get_table_rows_cell_values(table_body)
+        assert ["File name", file_name] in table_rows_cell_values
+
+        assert ["series_name", "series_y"] not in table_rows_cell_values
```

Comment:
dont quite understand the intention of this assertion, the data that we mock doesnt include a "series_y" so this would inevitably true. same with cref2.

You could change it to series_x and cref if you wanted to show that those aren't in it. Alternatively you could just assert the exact response of what table_rows_cell_values looks like 
e.g. `assert expected == table_rows_cell_values`